config/sync/core.extension.yml

```diff
@@ -15,7 +15,6 @@ module:
   ckeditor: 0
   ckeditor_a11ychecker: 0
   ckeditor_balloonpanel: 0
-  ckeditor_fixed_toolbar: 0
```

Comment:
Where is this functionality going?

Reply:
It doesn't even work at the moment from what I found. I'm thinking it'll be better suited for stanford_profile_helper https://stanfordits.atlassian.net/browse/D8CORE-1420

Reply:
It was working for me on `/node/add/stanford_event`

Reply:
https://devguide.sites.stanford.edu/node/add/stanford_event

Reply:
Hmm. Maybe just the basic page its not working then. I guess the question is, do we do the fork thing in drupal.org to support a non-stable module, or do we move the work into the profile helper module.

Reply:
I'm not familiar with the issues this module is causing but for what functionality it does offer, I suspect we can do without the trouble. Sounds like you have some custom work to do to get it to work in the react modal.